src/country.c

```diff
@@ -529,6 +530,7 @@ country_to_str(country c)
         case 248: return create_string(CONST_STRING("za"));
         case 249: return create_string(CONST_STRING("zm"));
         case 250: return create_string(CONST_STRING("zw"));
+        case 251: return create_string(CONST_STRING("xk"));
```

Comment:
this would need additional logic to have proper sorting.
Seeing `xk` after `zw` would probably be confusing.